de.fu_berlin.inf.dpp.ui.frontend/html/src/components/MainView/ActiveAccount.jsx

```diff
@@ -0,0 +1,29 @@
+import { Text } from 'react-localize'
+import React from 'react'
+
+class ActiveAccount extends React.Component {
+  render () {
+    const { activeAccount } = this.props
+    var accountText
```

Comment:
I would recommend using `let` instead of `var` whenever possible since it is [less error prone](https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var-to-declare-a-variable-in-jav)